be/src/runtime/exec_env.h

```diff
@@ -173,16 +175,20 @@ class ExecEnv {
     ClientCache<FrontendServiceClient>* _frontend_client_cache = nullptr;
     ClientCache<TPaloBrokerServiceClient>* _broker_client_cache = nullptr;
     ClientCache<TExtDataSourceServiceClient>* _extdatasource_client_cache = nullptr;
-    std::shared_ptr<MemTracker> _mem_tracker;
-    PoolMemTrackerRegistry* _pool_mem_trackers = nullptr;
+    // The ancestor of all trackers in the process. It is the only child of the root tracker.
+    // All manually created trackers should specify the process tracker as the parent.
+    std::shared_ptr<MemTracker> _process_mem_tracker = nullptr;
+    // The ancestor for all querys tracker.
+    std::shared_ptr<MemTracker> _all_query_mem_tracker = nullptr;
+    QueryMemTrackerRegistry* _query_mem_tracker_registry = nullptr;
```

Comment:
Why not use unique_ptr but use a raw pointer

Reply:
Thats great